hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmMetrics.java

```diff
@@ -367,6 +376,13 @@ public void testAclOperations() throws IOException {
 
   @Test
   public void testAclOperationsHA() throws Exception {
+    // This test needs a cluster with DNs and SCM to wait on safemode
+    cluster.shutdown();
```

Comment:
This looks a bit strange.  Would it be better to get rid of `@Before` and explicitly create cluster with desired properties for each test?  (Similar to what `TestOzoneManagerListVolumes` [did before](https://github.com/apache/ozone/blob/aae9ebd948e2b57bb947446d3b8fda9969cd6d47/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumes.java#L197-L200) HDDS-5717.)

Reply:
Yea I think you are correct. I have pushed a new commit that refactors things to ensure we only get the cluster we need for each test.